src/sources/android.ts

```diff
@@ -0,0 +1,5 @@
+import { isAndroid } from '../utils/browser'
+
+export default function collectIsAndroid(): boolean {
+  return isAndroid()
+}
```

Comment:
Idea: these new little files are not necessary. You can add `isAndroid` directly to the dictionary in `src/sources/index.ts`.

Reply:
I'm wondering if moving [getDocumentFocus](https://github.com/fingerprintjs/BotD/blob/main/src/utils/browser.ts#L119) out of utils into a source would make more sense.

Also `getMozAppearanceSupport`, `isIPad` and `isDesktopSafari` are unused apart from the debug info in the playground.

Reply:
I meant this:
```diff
- documentFocus: function hasDocumentFocus() {
-    return getDocumentFocus()
-  },
+ documentFocus: getDocumentFocus,
```

Reply:
🤦 Yes of course.

Reply:
> I'm wondering if moving [getDocumentFocus](https://github.com/fingerprintjs/BotD/blob/main/src/utils/browser.ts?rgh-link-date=2023-09-11T12%3A49%3A24Z#L119) out of utils into a source would make more sense.
>
> Also getMozAppearanceSupport, isIPad and isDesktopSafari are unused apart from the debug info in the playground.

These are questions for @r-valitov and @xnerhu